### PR TITLE
Add new module azure_rm_datalakestore

### PR DESCRIPTION
**Pull request by next-davidduquehernandez**
##### SUMMARY
This module allows to deploy, update and delete an Azure Data Lake Store (A.K.A. Data Lake Gen1). 

##### ISSUE TYPE
- New Module Pull Request

##### COMPONENT NAME
- azure_rm_datalakestore
- azure_rm_datalakestore_info

##### ADDITIONAL INFORMATION
All dev info could be found at [https://docs.microsoft.com/en-us/rest/api/datalakestore/](https://docs.microsoft.com/en-us/rest/api/datalakestore/)

### CODE REVIEW

File: plugins/module_utils/azure_rm_common.py [hunk 1]
```diff
@@ -1252,6 +1256,19 @@ def lock_models(self):
         self.log("Getting lock models")
         return ManagementLockClient.models('2016-09-01')
 
+    @property
+    def datalake_store_client(self):
+        self.log('Getting datalake store client...')
+        if not self._datalake_store_client:
+            self._datalake_store_client = self.get_mgmt_svc_client(DataLakeStoreAccountManagementClient,
```
Comment (Fred-sun) [inline plugins/module_utils/azure_rm_common.py:1307]: 1264 and 1265 lines continuation line under-indented for visual indent

File: plugins/modules/azure_rm_datalakestore.py [hunk 2]
```diff
@@ -0,0 +1,807 @@
...
+    name=dict(type='str', required=True),
+    start_ip_address=dict(type='str', required=True),
+    end_ip_address=dict(type='str', required=True)
+)
+
+virtual_network_rules_item = dict(
+    name=dict(type='str', required=True),
+    subnet_id=dict(type='str', required=True)
+)
+
```
Comment (Fred-sun) [inline plugins/modules/azure_rm_datalakestore.py:436]: expected 2 blank lines, found 1

File: plugins/modules/azure_rm_datalakestore_info.py [hunk 3]
```diff
@@ -0,0 +1,468 @@
...
+EXAMPLES = '''
+  - name: Get Azure Data Lake Store info from resource group 'myResourceGroup' and name 'myDataLakeStore'
+    azure_rm_datalakestore_info:
+      resource_group: myResourceGroup
+      name: myDataLakeStore
+
+  - name: Get Azure Data Lake Store info from resource group 'myResourceGroup'
+    azure_rm_datalakestore_info:
+      resource_group: myResourceGroup
+
```
Comment (Fred-sun) [inline plugins/modules/azure_rm_datalakestore_info.py]: blank line contains whitespace

File: plugins/modules/azure_rm_datalakestore_info.py [hunk 4]
```diff
@@ -0,0 +1,468 @@
...
+                    description:
+                        - The Key Vault information for connecting to user managed encryption keys.
+                    type: complex
+                    contains:
+                        key_vault_resource_id:
+                            description:
+                                - The resource identifier for the user managed Key Vault being used to encrypt.
+                            type: str
+                            returned: always
+                            sample: /subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/myResourceGroup/providers/Microsoft.KeyVault/vaults/testkv
```
Comment (Fred-sun) [inline plugins/modules/azure_rm_datalakestore_info.py]: line too long (161 > 160 characters)

File: plugins/modules/azure_rm_datalakestore_info.py [hunk 5]
```diff
@@ -0,0 +1,468 @@
...
+        type:
+            description:
+                - The resource type.
+            returned: always
+            type: str
+            sample: Microsoft.DataLakeStore/accounts
+        virtual_network_rules:
+            description:
+                - The list of virtual network rules associated with this Data Lake Store account.
+            type: list
```
Comment (Fred-sun) [inline plugins/modules/azure_rm_datalakestore_info.py]: trailing whitespace

File: plugins/modules/azure_rm_datalakestore_info.py [hunk 6]
```diff
@@ -0,0 +1,468 @@
...
+                name:
+                    description:
+                        - The resource name.
+                    type: str
+                    sample: Rule Name
+                subnet_id:
+                    description:
+                        - The resource identifier for the subnet.
+                    type: str
+                    sample: /subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/myResourceGroup/providers/Microsoft.Network/virtualNetworks/vnet/subnets/default
```
Comment (Fred-sun) [inline plugins/modules/azure_rm_datalakestore_info.py]: line too long (175 > 160 characters)

File: plugins/modules/azure_rm_datalakestore_info.py [hunk 7]
```diff
@@ -0,0 +1,468 @@
...
+        self.results = dict(
+            changed=False,
+            datalake=[]
+        )
+
+        self.name = None
+        self.resource_group = None
+
+        super(AzureRMDatalakeStoreInfo, self).__init__(self.module_arg_spec,
+                                                        supports_tags=False)
```
Comment (Fred-sun) [inline plugins/modules/azure_rm_datalakestore_info.py]: continuation line over-indented for visual indent

File: plugins/modules/azure_rm_datalakestore_info.py [hunk 8]
```diff
@@ -0,0 +1,468 @@
...
+            provisioning_state=datalake_store_obj.provisioning_state,
+            state=datalake_store_obj.state,
+            tags=datalake_store_obj.tags,
+            trusted_id_providers=datalake_store_obj.trusted_id_providers,
+            trusted_id_provider_state=datalake_store_obj.trusted_id_provider_state,
+            type=datalake_store_obj.type,
+            virtual_network_rules=None
+        )
+
+        account_dict['firewall_rules']=list()
```
Comment (Fred-sun) [inline plugins/modules/azure_rm_datalakestore_info.py]: missing whitespace around operator

File: plugins/modules/azure_rm_datalakestore_info.py [hunk 9]
```diff
@@ -0,0 +1,468 @@
...
+        for rule in datalake_store_obj.firewall_rules:
+            rule_item = dict(
+                name=rule.name,
+                start_ip_address=rule.start_ip_address,
+                end_ip_address=rule.end_ip_address
+            )
+            account_dict['firewall_rules'].append(rule_item)
+
+        account_dict['virtual_network_rules']=list()
```
Comment (Fred-sun) [inline plugins/modules/azure_rm_datalakestore_info.py]: missing whitespace around operator

File: plugins/modules/azure_rm_datalakestore_info.py [hunk 10]
```diff
@@ -0,0 +1,468 @@
...
+        for vnet_rule in datalake_store_obj.virtual_network_rules:
+            vnet_rule_item = dict(
+                name=vnet_rule.name,
+                subnet_id=vnet_rule.subnet_id
+            )
+            account_dict['virtual_network_rules'].append(vnet_rule_item)
+
+        if datalake_store_obj.identity:
+            account_dict['identity']=dict(
```
Comment (Fred-sun) [inline plugins/modules/azure_rm_datalakestore_info.py]: missing whitespace around operator

File: plugins/modules/azure_rm_datalakestore_info.py [hunk 11]
```diff
@@ -0,0 +1,468 @@
...
+                principal_id=datalake_store_obj.identity.principal_id,
+                tenant_id=datalake_store_obj.identity.tenant_id
+            )
+
+        if datalake_store_obj.encryption_config.key_vault_meta_info:
+            account_dict['encryption_config'] = dict(
+                key_vault_meta_info = dict(
+                    key_vault_resource_id = datalake_store_obj.encryption_config.key_vault_meta_info.key_vault_resource_id,
+                    encryption_key_name = datalake_store_obj.encryption_config.key_vault_meta_info.encryption_key_name,
+                    encryption_key_version = datalake_store_obj.encryption_config.key_vault_meta_info.encryption_key_version
```
Comment (Fred-sun) [inline plugins/modules/azure_rm_datalakestore_info.py]: 437,438,439,440 lines unexpected spaces around keyword / parameter equals

File: plugins/modules/azure_rm_datalakestore_info.py [hunk 12]
```diff
@@ -0,0 +1,468 @@
...
+                )
+            )
+
```
Comment (Fred-sun) [inline plugins/modules/azure_rm_datalakestore_info.py]: blank line contains whitespace

File: plugins/modules/azure_rm_datalakestore_info.py [hunk 13]
```diff
@@ -0,0 +1,468 @@
...
+            id=datalake_store_obj.id,
+            last_modified_time=datalake_store_obj.last_modified_time,
+            location=datalake_store_obj.location,
+            name=datalake_store_obj.name,
+            provisioning_state=datalake_store_obj.provisioning_state,
+            state=datalake_store_obj.state,
+            tags=datalake_store_obj.tags,
+            type=datalake_store_obj.type
+        )
+
```
Comment (Fred-sun) [inline plugins/modules/azure_rm_datalakestore_info.py:457]: expected 2 blank lines, found 1

**Review by Fred-sun — COMMENTED** [review]
@next-davidduquehernandez You can check the module through the link below. Thank you very much! link: https://docs.ansible.com/ansible/devel/dev_guide/testing_sanity.html#testing-sanity

File: plugins/module_utils/azure_rm_common.py (hunk 1 above)
Comment (next-davidduquehernandez) [reply plugins/module_utils/azure_rm_common.py:1307]: Done, thx!

File: plugins/modules/azure_rm_datalakestore.py (hunk 2 above)
Comment (next-davidduquehernandez) [reply plugins/modules/azure_rm_datalakestore.py:436]: Done, thx!

File: plugins/modules/azure_rm_datalakestore_info.py (hunk 3 above)
Comment (next-davidduquehernandez) [reply plugins/modules/azure_rm_datalakestore_info.py]: Done, thx!

File: plugins/modules/azure_rm_datalakestore_info.py (hunk 4 above)
Comment (next-davidduquehernandez) [reply plugins/modules/azure_rm_datalakestore_info.py]: Done, thx!

File: plugins/modules/azure_rm_datalakestore_info.py (hunk 5 above)
Comment (next-davidduquehernandez) [reply plugins/modules/azure_rm_datalakestore_info.py]: Done, thx!

File: plugins/modules/azure_rm_datalakestore_info.py (hunk 9 above)
Comment (next-davidduquehernandez) [reply plugins/modules/azure_rm_datalakestore_info.py]: Done, thx!

File: plugins/modules/azure_rm_datalakestore_info.py (hunk 10 above)
Comment (next-davidduquehernandez) [reply plugins/modules/azure_rm_datalakestore_info.py]: Done, thx!

File: plugins/modules/azure_rm_datalakestore_info.py (hunk 11 above)
Comment (next-davidduquehernandez) [reply plugins/modules/azure_rm_datalakestore_info.py]: Done, thx!

File: plugins/modules/azure_rm_datalakestore_info.py (hunk 12 above)
Comment (next-davidduquehernandez) [reply plugins/modules/azure_rm_datalakestore_info.py]: Done, thx!

File: plugins/modules/azure_rm_datalakestore_info.py (hunk 13 above)
Comment (next-davidduquehernandez) [reply plugins/modules/azure_rm_datalakestore_info.py:457]: Done, thx!

File: plugins/modules/azure_rm_datalakestore.py [hunk 14]
```diff
@@ -0,0 +1,808 @@
...
+                    - The resource identifier for the subnet.
+                type: str
+                required: true
+
+extends_documentation_fragment:
+    - azure.azcollection.azure
+    - azure.azcollection.azure_tags
+
+author:
+    David Duque Hernández (@next-davidduquehernandez)
```
Comment (Fred-sun) [inline plugins/modules/azure_rm_datalakestore.py]:
```suggestion
    - David Duque Hernández (@next-davidduquehernandez)
```

Comment (next-davidduquehernandez) [reply plugins/modules/azure_rm_datalakestore.py]: Done, thx!

File: plugins/modules/azure_rm_datalakestore_info.py [hunk 15]
```diff
@@ -0,0 +1,469 @@
...
+                    description:
+                        - The type of encryption being used.
+                    type: str
+                    sample: SystemAssigned
+                name:
+                    description:
+                        - The principal identifier associated with the encryption.
+                    type: str
+                    sample: 00000000-0000-0000-0000-000000000000
+                name:
```
Comment (Fred-sun) [inline plugins/modules/azure_rm_datalakestore_info.py]: duplicate with 188 line.

File: plugins/modules/azure_rm_datalakestore.py [hunk 16]
```diff
@@ -0,0 +1,808 @@
...
+                    description:
+                        - The type of encryption being used.
+                    type: str
+                    sample: SystemAssigned
+                name:
+                    description:
+                        - The principal identifier associated with the encryption.
+                    type: str
+                    sample: 00000000-0000-0000-0000-000000000000
+                name:
```
Comment (Fred-sun) [inline plugins/modules/azure_rm_datalakestore.py]: duplicate with line 314

**Review by Fred-sun — COMMENTED** [review]
@next-davidduquehernandez Please remove the duplicate lines!

File: plugins/modules/azure_rm_datalakestore.py [hunk 17]
```diff
@@ -0,0 +1,808 @@
...
+extends_documentation_fragment:
+    - azure.azcollection.azure
+    - azure.azcollection.azure_tags
+
+author:
+    - David Duque Hernández (@next-davidduquehernandez)
+'''
+
+EXAMPLES = '''
+  - name: Get Azure Data Lake Store info
```
Comment (Fred-sun) [inline plugins/modules/azure_rm_datalakestore.py]:
```suggestion
  - name: Create Azure Data Lake Store
```

**Comment by Fred-sun** [comment]
@next-davidduquehernandez Please specify location for the test case, such as "location: eastus2", because some location can't create date store, for example "location: eastus". Thank you very much!

**Comment by Fred-sun** [comment]
@next-davidduquehernandez Please specify the test case location parameter, such as "location: eastus2", because some location can't create date store, for example "location: eastus". Thank you very much!

File: plugins/modules/azure_rm_datalakestore_info.py (hunk 15 above)
Comment (next-davidduquehernandez) [reply plugins/modules/azure_rm_datalakestore_info.py]: It was a mistake with copy and paste. I setted with correct name. Thanks!

File: plugins/modules/azure_rm_datalakestore.py (hunk 16 above)
Comment (next-davidduquehernandez) [reply plugins/modules/azure_rm_datalakestore.py]: It was a mistake with copy and paste. I setted with correct name. Thanks!

File: plugins/modules/azure_rm_datalakestore.py (hunk 17 above)
Comment (next-davidduquehernandez) [reply plugins/modules/azure_rm_datalakestore.py]: Changed name to the proposal. Thanks!

**Comment by Fred-sun** [comment]
@next-davidduquehernandez Thank you for your contribution, please help to fix the plugins/module_utils/azure_rm_common.py file conflict! Thank you very much!

**Comment by next-davidduquehernandez** [comment]
@Fred-sun Merge realized, sanity and integration tests passed correctly after merge. Regards

File: plugins/module_utils/azure_rm_common.py [hunk 18]
```diff
@@ -1268,6 +1271,18 @@ def recovery_services_backup_client(self):
...
+                                                                   base_url=self._cloud_environment.endpoints.resource_manager,
+                                                                   api_version='2016-11-01')
+        return self._datalake_store_client
+
+    @property
+    def datalake_store_models(self):
+        return DataLakeStoreAccountModel
```
Comment (Fred-sun) [inline plugins/module_utils/azure_rm_common.py:1315]: expected 2 blank lines, found 1

**Comment by Fred-sun** [comment]
@next-davidduquehernandez Did you test the PlayBook you wrote? Can you share the results? Thank you!

**Comment by next-davidduquehernandez** [comment]
> @next-davidduquehernandez Did you test the PlayBook you wrote? Can you share the results? Thank you!

Hi @Fred-sun, of course.

My playbook:

```
---
- hosts: localhost
  connection: local

  collections:
    - azure.azcollection

  vars:
    base_dir: "{{ lookup('env','PWD') | basename }}"
    location: westeurope

  pre_tasks:

    - name: Set resource group
      set_fact:
        resource_group: "aaa"

  tasks:

    - name: Create resource group '{{ resource_group }}'
      azure_rm_resourcegroup:
        name: "{{ resource_group }}"
        location: "{{ location }}"

    - block:

        - name: Import storage test task
          import_tasks: "{{ lookup('env','PWD') }}/tests/integration/targets/azure_rm_datalakestore/tasks/main.yml"

      always:

        - name: Destroy resource group '{{ resource_group }}'
          azure_rm_resourcegroup:
            name: "{{ resource_group }}"
            location: "{{ location }}"
            force_delete_nonempty: true
            state: absent
```

Results:
```
ansible-playbook testing/playbook.yml 
[WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'

PLAY [localhost] ***************************************************************************************************************************************************************************

TASK [Gathering Facts] *********************************************************************************************************************************************************************
ok: [localhost]

TASK [Set resource group] ******************************************************************************************************************************************************************
ok: [localhost]

TASK [Create resource group 'aaa'] *********************************************************************************************************************************************************
changed: [localhost]

TASK [Create data lake store name] *********************************************************************************************************************************************************
ok: [localhost]

TASK [Create virtual network] **************************************************************************************************************************************************************
changed: [localhost]

TASK [Create subnet] ***********************************************************************************************************************************************************************
changed: [localhost]

TASK [Create minimal data lake store] ******************************************************************************************************************************************************
changed: [localhost]

TASK [Assert status succeeded and results] *************************************************************************************************************************************************
ok: [localhost] => {
    "changed": false,
    "msg": "All assertions passed"
}

TASK [Create minimal data lake store (Idempotence)] ****************************************************************************************************************************************
ok: [localhost]

TASK [Assert that status has not changed] **************************************************************************************************************************************************
ok: [localhost] => {
    "changed": false,
    "msg": "All assertions passed"
}

TASK [Update data lake store to add virtual_network_rules] *********************************************************************************************************************************
changed: [localhost]

TASK [Assert status succeeded and results include virtual_network_rules] *******************************************************************************************************************
ok: [localhost] => {
    "changed": false,
    "msg": "All assertions passed"
}

TASK [Update data lake store to change encryption state that must fail] ********************************************************************************************************************
fatal: [localhost]: FAILED! => {"changed": false, "msg": "Encryption type cannot be updated."}
...ignoring

TASK [Assert that encryption state cannot change] ******************************************************************************************************************************************
ok: [localhost] => {
    "changed": false,
    "msg": "All assertions passed"
}

TASK [Update data lake store to add new_tier] **********************************************************************************************************************************************
changed: [localhost]

TASK [Assert status succeeded and results include virtual_network_rules] *******************************************************************************************************************
ok: [localhost] => {
    "changed": false,
    "msg": "All assertions passed"
}

TASK [Delete minimal data lake store] ******************************************************************************************************************************************************
changed: [localhost]

TASK [Create new data lake store] **********************************************************************************************************************************************************
changed: [localhost]

TASK [Assert status succeeded and results include an Id value] *****************************************************************************************************************************
ok: [localhost] => {
    "changed": false,
    "msg": "All assertions passed"
}

TASK [Create new data lake store (Idempotence)] ********************************************************************************************************************************************
ok: [localhost]

TASK [Assert that status has not changed] **************************************************************************************************************************************************
ok: [localhost] => {
    "changed": false,
    "msg": "All assertions passed"
}

TASK [Delete virtual network] **************************************************************************************************************************************************************
changed: [localhost]

TASK [Delete Data Lake Store] **************************************************************************************************************************************************************
changed: [localhost]

TASK [Destroy resource group 'aaa'] ********************************************************************************************************************************************************
changed: [localhost]

PLAY RECAP *********************************************************************************************************************************************************************************
localhost                  : ok=24   changed=11   unreachable=0    failed=0    skipped=0    rescued=0    ignored=1
```

File: plugins/module_utils/azure_rm_common.py (hunk 18 above)
Comment (next-davidduquehernandez) [reply plugins/module_utils/azure_rm_common.py:1315]: Done

File: plugins/modules/azure_rm_datalakestore.py [hunk 19]
```diff
@@ -0,0 +1,808 @@
...
+#
+# Copyright (c) 2020 David Duque Hernández, (@next-davidduquehernandez)
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
```
Comment (Fred-sun) [inline plugins/modules/azure_rm_datalakestore.py]: Please remove the METADATA block it is no longer needed!

File: plugins/modules/azure_rm_datalakestore_info.py [hunk 20]
```diff
@@ -0,0 +1,469 @@
...
+#
+# Copyright (c) 2020 David Duque Hernández, (@next-davidduquehernandez)
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
```
Comment (Fred-sun) [inline plugins/modules/azure_rm_datalakestore_info.py]: Please remove the METADATA block it is no longer needed!

File: plugins/modules/azure_rm_datalakestore.py [hunk 21]
```diff
@@ -0,0 +1,808 @@
...
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+module: azure_rm_datalakestore
+version_added: "1.2.0"
```
Comment (Fred-sun) [inline plugins/modules/azure_rm_datalakestore.py]:
```suggestion
version_added: "1.4.0"
```

File: plugins/modules/azure_rm_datalakestore_info.py [hunk 22]
```diff
@@ -0,0 +1,469 @@
...
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_datalakestore_info
+version_added: "1.2.0"
```
Comment (Fred-sun) [inline plugins/modules/azure_rm_datalakestore_info.py]:
```suggestion
version_added: "1.4.0"
```

File: plugins/modules/azure_rm_datalakestore.py [hunk 23]
```diff
@@ -0,0 +1,808 @@
...
+            - Commitment_100TB
+            - Commitment_500TB
+            - Commitment_1PB
+            - Commitment_5PB
+        type: str
+    resource_group:
+        description:
+            - The name of the Azure resource group to use.
+        required: true
+        type: str
```
Comment (Fred-sun) [inline plugins/modules/azure_rm_datalakestore.py:132]:
```suggestion
        type: str
        aliases:
            - resource_group_name
```

File: plugins/modules/azure_rm_datalakestore_info.py [hunk 24]
```diff
@@ -0,0 +1,469 @@
...
+short_description: Get Azure Data Lake Store info
+description:
+    - Get Azure Data Lake Store info.
+
+options:
+    resource_group:
+        description:
+            - The name of the Azure resource group.
+        type: str
```
Comment (Fred-sun) [inline plugins/modules/azure_rm_datalakestore_info.py:23]:
```suggestion
        type: str
        aliases:
            - resource_group_name
```

**Review by Fred-sun — COMMENTED** [review]
Consistent with the argument_spec definition!

**Comment by Fred-sun** [comment]
@next-davidduquehernandez Please add this line "plugins/modules/azure_rm_datalakestore.py validate-modules:nonexistent-parameter-documented" to "tests/sanity/ignore-2.9.txt". Thanks!

File: plugins/modules/azure_rm_datalakestore.py (hunk 19 above)
Comment (next-davidduquehernandez) [reply plugins/modules/azure_rm_datalakestore.py]: Done

File: plugins/modules/azure_rm_datalakestore_info.py (hunk 20 above)
Comment (next-davidduquehernandez) [reply plugins/modules/azure_rm_datalakestore_info.py]: Done

File: plugins/modules/azure_rm_datalakestore.py (hunk 21 above)
Comment (next-davidduquehernandez) [reply plugins/modules/azure_rm_datalakestore.py]: Done

File: plugins/modules/azure_rm_datalakestore_info.py (hunk 22 above)
Comment (next-davidduquehernandez) [reply plugins/modules/azure_rm_datalakestore_info.py]: Done

File: plugins/modules/azure_rm_datalakestore.py (hunk 23 above)
Comment (next-davidduquehernandez) [reply plugins/modules/azure_rm_datalakestore.py:132]: Done

File: plugins/modules/azure_rm_datalakestore_info.py (hunk 24 above)
Comment (next-davidduquehernandez) [reply plugins/modules/azure_rm_datalakestore_info.py:23]: Done

**Comment by next-davidduquehernandez** [comment]
> @next-davidduquehernandez Please add this line "plugins/modules/azure_rm_datalakestore.py validate-modules:nonexistent-parameter-documented" to "tests/sanity/ignore-2.9.txt". Thanks!

Hi @Fred-sun, all modifications are realized. Thanks!

File: tests/integration/targets/azure_rm_datalakestore/aliases [hunk 25]
```diff
@@ -0,0 +1,3 @@
+cloud/azure
+shippable/azure/group2
+destructive
```
Comment (Fred-sun) [inline tests/integration/targets/azure_rm_datalakestore/aliases:3]: @next-davidduquehernandez The datalake resource only can create in some location, Please changed as follow! Thank you much!
```suggestion
destructive
disabled
```

**Review by Fred-sun — COMMENTED** [review]
The changed is OK, Please make last change!

**Pull request merged**